docs/using-overseerr/notifications/email.md

```diff
@@ -0,0 +1,39 @@
+# Email
+
+{% hint style="info" %}
+**Media Requested**, **Media Automatically Approved**, and **Media Failed** email notifications are sent to _all_ users with the **Manage Requests** permission, as these notification types are intended for application administrators rather than end users.
+
+**Media Approved**, **Media Declined**, and **Media Available** email notifications are sent to the user who submitted the request.
+{% endhint %}
+
+## Configuration
+
+## Sender Address (required)
+
+Set this to the email address you would like to appear in the "from" field of the email message.
+
+## Sender Name (optional)
+
+Configure a friendly name for the email sender.
+
+## SMTP Host
+
+Set this to the hostname or IP address of your SMTP host/server.
+
+## SMTP Port
+
+Set this to a supported port number for your SMTP host. `465` and `587` are commonly used.
+
+## Enable SSL (optional)
+
+If using an SMTP serves which supports [STARTTLS](https://en.wikipedia.org/wiki/Opportunistic_TLS), this should **not** be enabled. If you configured the SMTP port as `587`, you should leave this unchecked in most cases.
+
+This setting should only be enabled to establish secure connections for hosts which do not support STARTTLS.
+
+## SMTP Username & Password
+
+Configure these values as appropriate to authenticate with your SMTP host.
+
```

Comment:
add...
```
{% hint style="info" %}
If your account requires 2FA you may be required to create an app password instead of using your account password. 
{% endhint %}
```